Makefile

```diff
@@ -43,6 +43,8 @@ dae: ebpf
 clean-ebpf: 
 	@rm -f control/bpf_bpf*.go && \
 		rm -f control/bpf_bpf*.o
+fmt:
+	go fmt ./...
```

Comment:
I've integrated this automation in `pre-commit`. Feel free to have a try afterward.